eth/backend.go

```diff
@@ -590,6 +592,11 @@ func (s *Ethereum) Protocols() []p2p.Protocol {
 	if s.config.SnapshotCache > 0 {
 		protos = append(protos, snap.MakeProtocols((*snapHandler)(s.handler), s.snapDialCandidates)...)
 	}
+
+	if s.config.EnableFastFinality {
+		protos = append(protos, ronin.MakeProtocols((*roninHandler)(s.handler))...)
```

Comment:
should we move checking `EnableFastFinality` inside the protocol? So that we can scale further in the future where we might have other logics that needed to send via `ronin` protocol?

Reply:
Okay, let me try to decouple ronin protocol and fast finality

Reply:
I added another flags for configuring ronin protocol